### ISSUE TITLE
Aperture plotting for the keyword CHAMBER

### ISSUE THREAD

**Issue by rtesse** [chernals/zgoubido#35]
We should find a way to plot the aperture for the keyword CHAMBER. The definition of a CHAMBER is as follow:

```
Chamber(IA=2),
Chamber('M1_slits', IA=1, IFORM=2, J=0, C1=29.75 * _.mm, C2=29.75 * _.mm),
Drift('DM1', XL=30*_.cm),
Chamber(IA=2),
```
Therefore, there is no explicit length to the element.

**Comment by chernals** [comment]
A few thoughts:

- the aperture is a "global" property of the line (because it is not "local" in the sense that it is not the property of a single keyword but comes from these on/off flags)
- as such I think that we should treat that in the survey phase
- we could/should add a property to every element (or "drawable" element or "patchable" or something like that) set at zero by default
- during the survey we change that property as we loop over the line (so we need a bunch of properties, like iform, c1, c2, etc)
- then easy peacy when we draw the line

What do you think ?

**Comment by rtesse** [comment]
I agree; we can do it that way. However, we will probably have to change the way the elements are plotted to take this into account, but it should be possible.
